### PR TITLE
Switching embeds to v13 embeds and code improvements

### PR DESCRIPTION
**Pull request by xstarrystar**
As mentioned in the title and also switching to node v16 because why not.

### CODE REVIEW

**Comment by Revenge58** [comment]
I improved **some** codes not all.

File: Commands/Fun/country.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
         const country = args.slice().join(' ');
         if (!country) {
             return message.channel.send(
-                ':x: Please provide a valid Country.',
+                ':x: Please provide a valid Country.',A
```
Comment (sujalgoel) [inline Commands/Fun/country.js:17]: What is that "A" for?

File: Commands/Anime/kiss.js [hunk 2]
```diff
@@ -30,26 +30,10 @@ module.exports = {
...
-                        .setColor(config.embedcolor)
-                        .setDescription( `${message.member} kisses ${message.mentions.members.first()}!`)
-                        .setImage(url)
-                );
-            } else {
-                return message.channel.send(
-                    `Sorry ${message.member}, I can't seem to locate your imaginary friend.`
-                );
-            }
+            if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`S~seriously?!`);
```
Comment (sujalgoel) [inline Commands/Anime/kiss.js:34]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/kiss.js [hunk 3]
```diff
@@ -30,26 +30,10 @@ module.exports = {
...
+            if (message.mentions.members.size) return message.channel.send({ embeds: [embed.setColor(config.embedcolor).setDescription( `${message.member} kisses ${message.mentions.members.first()}!`).setImage(url)]});
+            return message.channel.send(`Sorry ${message.member}, I can't seem to locate your imaginary friend.`);
```
Comment (sujalgoel) [inline Commands/Anime/kiss.js:36]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/poke.js [hunk 4]
```diff
@@ -10,36 +10,12 @@ module.exports = {
...
-                    .setColor(config.embedcolor)
-                    .setDescription(`${message.member} pokes ${message.mentions.members.first()}!`)
-                    .setImage(url)
-            );
-        } else {
-            return message.channel.send(
-                `${message.member}, I can't poke your imaginary friend! :(`
-            );
-        }
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`${message.member}, I'm already here! You need something?`);
```
Comment (sujalgoel) [inline Commands/Anime/poke.js:16]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/poke.js [hunk 5]
```diff
@@ -10,36 +10,12 @@ module.exports = {
...
+        if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`What?`);
```
Comment (sujalgoel) [inline Commands/Anime/poke.js:17]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/poke.js [hunk 6]
```diff
@@ -10,36 +10,12 @@ module.exports = {
...
-            );
-        } else {
-            return message.channel.send(
-                `${message.member}, I can't poke your imaginary friend! :(`
-            );
-        }
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`${message.member}, I'm already here! You need something?`);
+        if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`What?`);    
+        if (message.mentions.members.size) return message.channel.send({ embeds: [embed.setColor(config.embedcolor).setDescription(`${message.member} pokes ${message.mentions.members.first()}!`).setImage(url)]});
+        return message.channel.send(`${message.member}, I can't poke your imaginary friend! :(`);
```
Comment (sujalgoel) [inline Commands/Anime/poke.js:19]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/slap.js [hunk 7]
```diff
@@ -11,37 +11,11 @@ module.exports = {
...
-          return message.channel.send(`Wai~ Seriously!?`);
-        } else if (message.mentions.members.size) {
-          return message.channel.send(
-            embed
-              .setColor(config.embedcolor)
-              .setDescription(`${message.member} slapped ${message.mentions.members.first()}!`)
-              .setImage(url)
-          );
-        } else {
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`${[`Ouch! How dare you slap me!`, `Stop that!`, `It hurts! ;-;`][Math.floor(Math.random() * 2)]}`);
```
Comment (sujalgoel) [inline Commands/Anime/slap.js:16]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/slap.js [hunk 8]
```diff
@@ -11,37 +11,11 @@ module.exports = {
...
+         if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`Wai~ Seriously!?`);
```
Comment (sujalgoel) [inline Commands/Anime/slap.js:17]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/tickle.js [hunk 9]
```diff
@@ -12,34 +12,11 @@ module.exports = {
...
-                    .setColor(config.embedcolor)
-                    .setDescription(`${message.member} started tickling ${message.mentions.members.first()}!`)
-                    .setImage(url)
-            );
-        } else {
-            return message.channel.send(
-                embed.setColor(config.embedcolor).setImage(url)
-            );
-        }
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`B~Baka ${message.member}! Stop that~ it tickles!`);
```
Comment (sujalgoel) [inline Commands/Anime/tickle.js:17]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Anime/tickle.js [hunk 10]
```diff
@@ -12,34 +12,11 @@ module.exports = {
...
-                    .setDescription(`${message.member} started tickling ${message.mentions.members.first()}!`)
-                    .setImage(url)
-            );
-        } else {
-            return message.channel.send(
-                embed.setColor(config.embedcolor).setImage(url)
-            );
-        }
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`B~Baka ${message.member}! Stop that~ it tickles!`); 
+        if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`Wai~ Seriously!?`);
```
Comment (sujalgoel) [inline Commands/Anime/tickle.js:18]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Chatbot/disablechatbotchannel.js [hunk 11]
```diff
@@ -30,19 +30,19 @@ module.exports = {
                 let a = db.fetch(`chatbot_${message.guild.id}`);
 
                 if(!a) {
-                    return message.channel.send({embed: {
+                    return message.channel.send({embeds: [{
```
Comment (sujalgoel) [inline Commands/Chatbot/disablechatbotchannel.js:33]: I would prefer the use of MessageEmbed here

File: Commands/Chatbot/setchatbotchannel.js [hunk 12]
```diff
@@ -32,10 +32,10 @@ module.exports = {
                 if(message.guild.channels.cache.has(b)) {
                         return message.channel.send(`**${emotes.verified} ChatBot Channel Set In This Server Is \`${channelName.name}\`!**`);
                 } else {
-                    return message.channel.send({embed: {
+                    return message.channel.send({embeds: [{
```
Comment (sujalgoel) [inline Commands/Chatbot/setchatbotchannel.js:35]: I would prefer the user of MessageEmbed instead of this

File: Commands/Fun/clyde.js [hunk 13]
```diff
@@ -9,16 +9,11 @@ module.exports = {
         category: __dirname.split("Commands\\")[1]
     },
     run: async (client, message, args) => {
-
         const text = args.slice().join(' ');
         if (!text) {
-            return message.channel.send(
-                '❎ Please provide valid text.',
-            );
+            return message.channel.send('❎ Please provide valid text.');
```
Comment (sujalgoel) [inline Commands/Fun/clyde.js:14]: You should use 
```js
message.channel.send({content: ""})
```

File: Commands/Fun/country.js (hunk 1 above)
Comment (sujalgoel) [inline Commands/Fun/country.js:17]: What is the use of "A" here?

File: Commands/Nsfw/bondage.js [hunk 14]
```diff
@@ -10,10 +10,10 @@ module.exports = {
     },
     run: async (client, message, args) => {
         if (!message.channel.nsfw) return message.channel.send({
-            embed: {
+            embeds: [{
```
Comment (sujalgoel) [inline Commands/Nsfw/bondage.js:13]: You should use MessageEmbed here

File: Commands/Nsfw/hentai.js [hunk 15]
```diff
@@ -13,10 +13,10 @@ module.exports = {
         if (!message.channel.nsfw) {
             message.react('💢');
             return message.channel.send({
-                embed: {
+                embeds: [{
```
Comment (sujalgoel) [inline Commands/Nsfw/hentai.js:16]: You should use MessageEmbed here

**Review by sujalgoel — COMMENTED** [review]
Please see the comments and there isn't any big change or something. Also, try to use a linter for cleaning the code.

**Comment by sujalgoel** [comment]
The code is still working fine but we should make the changes and also remove that undefined variable "A" from some files.

File: Commands/Anime/kiss.js (hunk 2 above)
Comment (Revenge58) [reply Commands/Anime/kiss.js:34]: If it's string only, you can use message.channel.send("string")

File: Commands/Fun/country.js (hunk 1 above)
Comment (Revenge58) [reply Commands/Fun/country.js:17]: Probably a typo.

File: Commands/Anime/kiss.js (hunk 3 above)
Comment (Revenge58) [reply Commands/Anime/kiss.js:36]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/poke.js (hunk 4 above)
Comment (Revenge58) [reply Commands/Anime/poke.js:16]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/poke.js (hunk 5 above)
Comment (Revenge58) [reply Commands/Anime/poke.js:17]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/poke.js (hunk 6 above)
Comment (Revenge58) [reply Commands/Anime/poke.js:19]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/slap.js (hunk 7 above)
Comment (Revenge58) [reply Commands/Anime/slap.js:16]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/slap.js (hunk 8 above)
Comment (Revenge58) [reply Commands/Anime/slap.js:17]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/tickle.js (hunk 9 above)
Comment (Revenge58) [reply Commands/Anime/tickle.js:17]: If it's string only, you can use message.channel.send("string")

File: Commands/Anime/tickle.js (hunk 10 above)
Comment (Revenge58) [reply Commands/Anime/tickle.js:18]: If it's string only, you can use message.channel.send("string")

**Comment by LegendaryEmoji** [comment]
wut

**Comment by Revenge58** [comment]
> wut

shut

**Comment by Revenge58** [comment]
oh.

**Comment by Lebyy** [comment]
@DeltaCoderr I suggest you to start implementing slash commands in karma and this pr LGTM.

**Comment by Revenge58** [comment]
> @DeltaCoderr I suggest you to start implementing slash commands in karma other than that this pr LGTM.

Um, I think because of the chat-bot he'll have the message intent so there's no need to change to slash commands and discord said they're not forcing slash commands meaning if he gets the message intent he should have no problem.

**Comment by Lebyy** [comment]
> > @DeltaCoderr I suggest you to start implementing slash commands in karma other than that this pr LGTM.
> 
> Um, I think because of the chat-bot he'll have the message intent so there's no need to change to slash commands and discord said they're not forcing slash commands meaning if he gets the message intent he should have no problem.

I see alright.

**Comment by Lebyy** [comment]
Also about this [file](https://github.com/Revenge58/KarmaBot/blob/develop/node.sh) I suggest updating the npm and nvm version to latest.

**Comment by DeltaCoderr** [comment]
> Also about this [file](https://github.com/Revenge58/KarmaBot/blob/develop/node.sh) I suggest updating the npm and nvm version to latest.

alright

**Review by sbdh11 — APPROVED** [review]

**Comment by DeltaCoderr** [comment]
Thanks!

**Pull request merged**